2021q3/freebsd-foundation.adoc

```diff
@@ -0,0 +1,169 @@
+material to promote, educate, and advocate for the FreeBSD Project, facilitate
+collaboration between commercial vendors and FreeBSD developers, and finally,
+represent the FreeBSD Project in executing contracts, license agreements, and
+other legal arrangements that require a recognized legal entity.
+
+Here are some highlights of what we did to help FreeBSD last quarter:
+
+==== Fundraising Efforts
+
+Fundraising last quarter wasn’t as spectacular as we were hoping. But, then
+again, people tend to take vacations during the summer months, which makes it
+that more difficult for our funding requests to go through the management chain
+for approvals.
+
+So far this year we’ve raised $180,000 towards our $2,000,000 spending
+budget. Why do we need so much money? Well, last year we decided to make more
+significant software contributions to FreeBSD.  In order to do that, we had to
+grow our team.  We developed
+link:https://freebsdfoundation.org/blog/technology-roadmap/[a technology
+roadmap] based on input we were receiving from commercial users as well as
+market trends. Based on the roadmap, we identified positions we needed to fill.
+
+This year we've hired three full-time software developers, one full-time ARM
+kernel developer, and one project manager. We also are funding wifi work
+full-time and some other projects to help with FreeBSD on the desktop.  You can
+read about this effort to attract new users and contributors to the Project in
+individual entries elsewhere in this status report.
+
+Our growth wasn’t just in our technology team, but in our advocacy team
+too. Here we hired a marketing coordinator and technical writer to provide more
+educational and informational content. You’ll see in the *Advocacy and
+Education* section below all the work we did to promote FreeBSD, provide
+community engagement, education opportunities, and informative content to
+help pave the path to getting started with FreeBSD.
+
+You’ll find out how we used your donations here and in individual entries
+throughout this status report.
+
+We’re passionate about supporting you, the FreeBSD community, but we can’t do it
+without your financial support.
+
+Please consider making a donation to help us continue and increase our support
+for FreeBSD in 2021:
+link:https://www.FreeBSDfoundation.org/donate/[https://www.FreeBSDfoundation.org/donate/].
+
+We also have the Partnership Program, to provide more benefits for our larger
+commercial donors. Find out more information at
+link:https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/[https://www.FreeBSDfoundation.org/FreeBSD-foundation-partnership-program/]
+and share with your companies!
+
+==== OS Improvements
+
+During the third quarter, Foundation staff and grant recipients committed 420
+src tree changes, 24 ports tree changes, and 11 doc tree changes. This
+represents 38% of src commits which identify a sponsor. For doc and ports
+commits the values are 48% and 16%.
```

Comment:
Are these the sponsored commits/total commits %ages? Or Foundation-sponsored/total sponsored? Either way, maybe make it clearer?

Reply:
These are <Foundation Sponsored> / <Total Sponsored> - i.e., 38% of src commits which identify a sponsor are FF sponsored.

(FWIW, I think about 1/2 of src commits in that period are sponsored)

Reply:
Hopefully it's clearer now.